MissingPropertyException: No such property: antBuilderInput

I get the following Exception:

```java
Caused by: groovy.lang.MissingPropertyException: No such property: antBuilderInput for class: com.lv.plugins.weblogic.tasks.WLDeployTask_Decorated
	at org.gradle.api.internal.plugins.ExtraPropertiesDynamicObjectAdapter.setProperty(ExtraPropertiesDynamicObjectAdapter.java:46)
	at org.gradle.api.internal.CompositeDynamicObject.setProperty(CompositeDynamicObject.java:122)
	at com.lv.plugins.weblogic.tasks.WLDeployTask_Decorated.setProperty(Unknown Source)
	at com.lv.plugins.weblogic.tasks.AbstractWLDeployTask.getAntBuilderInput(AbstractWLDeployTask.groovy:152)
	at com.lv.plugins.weblogic.tasks.AbstractWLDeployTask.run(AbstractWLDeployTask.groovy:115)
```

What am i doing wrong?

--- a/demo/build.gradle
+++ b/demo/build.gradle
@@ -5,13 +5,13 @@ buildscript {
         if( project.hasProperty('internalArtifactRepository') ){
             maven { url project.internalArtifactRepository }
         } else {
-            maven { url 'http://dl.bintray.com/lv/gradle-plugins/' }
+            maven { url 'https://plugins.gradle.org/m2/' }
             mavenCentral()
         }
     }
 
     dependencies {
-        classpath 'com.lv.plugins:gradle-weblogic-plugin:0.3'
+        classpath 'gradle.plugin.com.lv.plugins:gradle-weblogic-plugin:0.4'
     }
 }
 

